string sn
uint64 timestamp
uint32 peer_id
string task_guid
DynamicPosition pos

# CarWorkState
uint8 car_work_state
uint8 CAR_UNKOWN = 0
uint8 CAR_READY = 1
uint8 CAR_RUNNING = 2
uint8 CAR_ERROR = 10

string description
string drone_sn